src/lib/features/feature-search/feature-search-store.ts

```diff
@@ -327,7 +332,15 @@ class FeatureSearchStore implements IFeatureSearchStore {
                 'ranked_features.feature_name',
                 'lifecycle.stage_feature',
             );
+
         if (this.flagResolver.isEnabled('flagsOverviewSearch')) {
+            const parsedLifecycle = lifecycle
+                ? parseSearchOperatorValue('lifecycle.latest_stage', lifecycle)
+                : null;
+            if (parsedLifecycle) {
+                applyGenericQueryParams(finalQuery, [parsedLifecycle]);
```

Comment:
the important part is that we can't add generic query param with other query params because we don't have lifecycle information in the ranked_features query. So I decided to add the extra query for lifecycle at the very end where the latest stage is already selected. Conveniently it can be covered by the same feature flag as the very last query for the change requests that the feature is used in.